doc/deoplete.txt

```diff
@@ -553,6 +553,7 @@ tag							*deoplete-source-tag*
 
 file							*deoplete-source-file*
 		This source collects keywords from local files.
+		omni
```

Comment:
Sorry about that, I believe I missclicked and hadn't seen that little bastard :). Please, see my last [commit](https://github.com/brooth/deoplete.nvim/commit/5d4cb8bfcd9ba752e94b283bb13f00b346eff00b).